ui/src/components/People/DetailsCard.vue

```diff
@@ -155,8 +160,22 @@
 import DynamicForm from "@/components/Form/DynamicForm.vue";
 import Practitioner from "@/mixins/Practitioner.js";
 import StructureDefinition from "@/mixins/StructureDefinition.js";
+import MobileLayout from "@/mixins/MobileLayout.js";
 
 export default {
+  computed:{
+    setDetailCardTitleStyle()
+    {
+      return this.detailCardTitleStyle(this.screenSize);
+    },
+    setDetailCardBtnSize()
+    {
+      var btnSize=this.detailCardBtnSize(this.screenSize);
+      console.log("Computed btn size: "+btnSize);
```

Comment:
Remove the console and the commented out section. Also, this method should just be btnSize.